exercises/grep/example.go

```diff
@@ -0,0 +1,102 @@
+package grep
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type grepOptions struct {
```

Comment:
since we're grep package, might be only options

Reply:
ok